.gitignore

```diff
@@ -11,3 +11,4 @@ lab/
 .vscode/
 .env
 .yarn
+modules/**/dist/
```

Comment:
Suprised that each coin module had to define this explicitly.... 

Only an issue now because the new coredao module doesn't have a gitignore, so I keep getting a `dist` in my un-staged files.